containers/resalloc-server.Containerfile

```diff
@@ -0,0 +1,41 @@
+# TODO: Check if we can use ubi.
+# resalloc dependencies require a subscription.
+# FROM registry.access.redhat.com/ubi9/ubi
+FROM quay.io/centos/centos:stream9
+USER 0
+
+RUN dnf install -y dnf-plugins-core
+RUN dnf config-manager --set-enabled crb extras-common
```

Comment:
Just a note;  the `dnf install -y` command here adds the DNF caches into the intermediary/cache image, the fact that you `dnf clean all` later has no effect on the overall image payload.

I think you want to squash all the DNF commands into a single `RUN` command.